app-arch/pixz/pixz-1.0.6-r1.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI=7
```

Comment:
This is not a proper EAPI=7 migration. Please read https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html -- there's more to do like setting BDEPEND.